chain/consensus/filcns/upgrades.go

```diff
@@ -1494,11 +1497,11 @@ func upgradeActorsV9Common(
 	root cid.Cid, epoch abi.ChainEpoch, ts *types.TipSet,
 	config nv17.Config,
 ) (cid.Cid, error) {
-	buf := blockstore.NewTieredBstore(sm.ChainStore().StateBlockstore(), blockstore.NewMemorySync())
-	store := store.ActorStore(ctx, buf)
+	writeStore := blockstore.NewAutobatch(ctx, sm.ChainStore().StateBlockstore(), units.GiB/4)
```

Comment:
I assume @Stebalien approved this

Reply:
I haven't seen this. Looking through it now.

Reply:
Oh, nvm. Yes, this is what we want. Otherwise, we buffer in memory and oom.